lib/timetable-form/Form.tsx

```diff
@@ -1,36 +1,57 @@
-import { useState } from "react";
-import "govuk-frontend/dist/govuk/govuk-frontend.min.css";
+import { useMemo, useState } from "react";
+import { v4 as uuidv4 } from "uuid";
 
 import { Button } from "../gds-components/button/Button";
 import { TextInput } from "../gds-components/text-input/TextInput";
 import { DateInput } from "../gds-components/date-input/DateInput";
-import { Stages, FormData } from "../types/timetable";
-import {
-  formStateToDevelopmentPlanTimetables,
-  devPlanToCSVString,
-} from "../utils/timetable";
+import { DevelopmentPlan } from "../types/timetable";
+import { devPlanToCSVString } from "../utils/timetable";
 import { stageNames } from "../constants";
 
 import styles from "./styles.module.css";
+import "govuk-frontend/dist/govuk/govuk-frontend.min.css";
```

Comment:
Out of interest, why did this import get moved to the bottom? I'm happy to go with whatever we like (and there are lint rules we can add) but my preference is to have imports ordered by how "far away" they are, with external imports at the top.

Reply:
I guess it's a preference of mine to have style imports last. I have ordered the imports in the other two components in the same fashion. Likewise I am also happy to go with whatever we agree on